lib/postgres.js

```diff
@@ -209,9 +210,10 @@ class Postgres {
             buf.writeFloatBE(item.floatvalue, 0);
             item.paintwear = buf.readInt32BE(0);
 
-            if (item.floatvalue <= 0 && item.defindex !== 507) {
-                // Only insert weapons, naive check
-                // Special case for the 0 float Karambit
+            if (item.floatvalue <= 0 && item.defindex !== 507 && item.defindex !== 1355) {
+                // Only insert weapons and keychains, naive check
+                // Special case for the 0 float Karambit and keychains.
+                // The keychains has a unique pattern
```

Comment:
We shouldn't include saving keychains to the local DB within this PR -- right now only skins are meant to be added.

A separate PR with a config entry to save certain entities like keychains/charms would be fine.